components/product/ProductCard/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { useOffer } from "$store/sdk/useOffer.ts";
+import type { Product } from "apps/commerce/types.ts";
+import { PriceAndName } from "deco-sites/persono/components/product/ProductCard/components/PriceAndName.tsx";
+import { useMemo } from "preact/compat";
+import { ProductCardImage } from "./components/ProductCardImage.tsx";
+import { SendEventOnClick } from "deco-sites/persono/components/Analytics.tsx";
+import { mapProductToAnalyticsItem } from "apps/commerce/utils/productToAnalyticsItem.ts";
+
+interface Props {
+  product: Product;
+  /** Preload card image */
+  preload?: boolean;
+
+  /** @description index of the product card in the list */
+  index?: number;
+
+  search?: boolean;
+
+  itemListName?: string;
+}
+
+const relative = (url: string) => {
+  const link = new URL(url);
+  return `${link.pathname}${link.search}`;
+};
+
+function ProductCard({ product, preload, index, search, itemListName }: Props) {
+  const { url, productID, name, image: images, offers } = product;
+  const { price = 0, installments, listPrice = 0 } = useOffer(offers);
+  const [front] = images ?? [];
+
+  const id = `product-card-${productID}`;
+
+  // Necessario atualizar quando as especificações dos produtos retorarem corretamente
+  const newsTitle = "Novidade";
+  const hasNews = false;
+
+  const { hasDiscount, discount, hasMultiplePrices } = useMemo(() => {
+    const variantPrices = product.isVariantOf?.hasVariant.map(
+      (item) => item.offers?.offers?.[0]?.price
+    );
+
+    const hasMultiplePrices =
+      variantPrices && variantPrices?.length > 2
```

Comment:
Se tiver mais de uma variant entao tem multiplePrices
```suggestion
      variantPrices && variantPrices?.length > 1
```